media_utils/video_puppeteer.py

```diff
@@ -187,12 +198,18 @@ def playback_done(video):
     # either still playing or there is a problem, so this will never
     # return True.
 
+    active_re = re.compile('NextGetSampleIndex:([\d]+)')
+
     debug_lines = video.get_debug_lines()
     if debug_lines is not None:
+        for occurence in active_re.findall(' '.join(debug_lines)):
+            if occurence.group(1) == '0':
+                raise VideoException('Video %s is stalled' % video)
+
         num_active = len(active_re.findall(' '.join(debug_lines)))
         if num_active == 2:
             return False
 
-    raise VideoException('Did not find exactly one audio and one video '
+    raise VideoException('Did not find video debug information for '
```

Comment:
* This also raises if num_active =! 2. Do we want that? 
* Modify exception message accordingly